resources/user-guides/Importing_Blockchain.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: static_page
+title: Importing the Monero Blockchain from an outside source 
+---
+
+# Importing the Blockchain to Monero GUI wallet (Windows)
+
+### Step 1
+
+Download the Current bootstrap from https://downloads.getmonero.org/blockchain.raw, you can skip this step if you are importing the Blockchain from another source.
+
+### Step 2
+
+Find the Path of your Monero wallet (the folder where you extracted your wallet) for example mine is.
+
+`D:\monero-gui-0.10.3.1-beta2`

```

Comment:
Latest release is monero-gui-0.10.3.1.zip (no beta2). Default zip action on Windows is to extract to folder named the same as zip. For consistency, possibly change this to match the most recent release's folder name that they are likely to see?